sdk/python/requirements/py3.8-ci-requirements.txt

```diff
@@ -78,7 +78,7 @@ babel==2.12.1
     # via sphinx
 backcall==0.2.0
     # via ipython
-backports-zoneinfo==0.2.1
+backports-zoneinfo==0.2.1;python_version<"3.9"
```

Comment:
Needed? I'm assuming these requirement files are auto generated.

Reply:
This is a necessary change to get py3.8 to compile.

I added this comment to the Feast dev guide: 
https://github.com/feast-dev/feast/issues/2105#issuecomment-1613906129

backports-zoneinfo is only present in the 3.8 requirements, but this package fails with Python 3.9 or higher. Without this requirement, the makefile may choose to attempt to pull a 3.9 version, and fail to compile.

Reply:
If you look at the ci-requirements.txt for py 3.9 and 3.10, you will not find that dependency. To install feast, we need to use `make install-python-ci-dependencies`

Reply:
It did not build for me when I removed the dependency. I'm not sure what it's covering though. I tend not to like removing items with being sure why I'm removing them.

Reply:
Makefile has a problem in getting your current version..PYTHON variable is not defined. Can you define the following variable in `PYTHON:=$(shellpython-c'importsys;print(str(sys.version_info[0])+"."+str(sys.version_info[1]))')` and run `make install-python-ci-dependencies` to try out

Reply:
Yes, it works for 3.10, which is what I have, but it didn't work for 3.8 until I fixed the above.

Reply:
I'll verify and update it if necessary. Thank you.